ivy_tests/test_ivy/test_misc/test_array.py

```diff
@@ -977,12 +986,11 @@ def test_array__rdivmod__(
 @handle_method(
     method_tree="Array.__truediv__",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("numeric"),
+        available_dtypes=helpers.get_dtypes("real_and_complex"),
```

Comment:
Why are we limiting the data type generation to real_and_complex?

Reply:
real_and_complex is numeric + complex so it's widening the data type